build/cqs.tsv

```diff
@@ -231,40 +230,40 @@ com.datastax.cassandra:cassandra-driver-core	3.0.0	provided
 com.datastax.cassandra:cassandra-driver-mapping	3.0.0	provided
 javax.media:jai_codec	1.1.3	provided
 javax.media:jai_core	1.1.3	provided
-org.apache.hadoop:hadoop-annotations	2.5.2	provided
-org.apache.hadoop:hadoop-client	2.5.2	provided
-org.apache.hadoop:hadoop-common	2.5.2	provided
-org.apache.hadoop:hadoop-distcp	2.5.2	provided
-org.apache.hadoop:hadoop-mapreduce-client-core	2.5.2	provided
-org.apache.hadoop:hadoop-yarn-api	2.5.2	provided
-org.apache.hadoop:hadoop-yarn-common	2.5.2	provided
+org.apache.hadoop:hadoop-annotations	2.6.4	provided
+org.apache.hadoop:hadoop-client	2.6.4	provided
+org.apache.hadoop:hadoop-common	2.6.4	provided
+org.apache.hadoop:hadoop-distcp	2.6.4	provided
+org.apache.hadoop:hadoop-mapreduce-client-core	2.6.4	provided
+org.apache.hadoop:hadoop-yarn-api	2.6.4	provided
+org.apache.hadoop:hadoop-yarn-common	2.6.4	provided
```

Comment:
done - should be covered under ~https://dev.eclipse.org/ipzilla/show_bug.cgi?id=8504~